LeaderboardBackend.Test/Features/Users/UserPasswordRuleTests.cs

```diff
@@ -0,0 +1,44 @@
+using FluentValidation;
+using FluentValidation.TestHelper;
+using LeaderboardBackend.Models.Validation;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test;
+
+public class UserPasswordTests
+{
+    private readonly TestValidator _sut = new();
+
+    [TestCase("MWqVR7vHLZvizTLQX7")]
+    [TestCase("5#SP:|2^U?2PGt.fv1", Description = "contains special characters")]
+    [TestCase("cA8RlO8i", Description = "8 characters (minimum")]
+    [TestCase("9cAf9eQIt3zSNCJFfGzAX5Fu2EjCxf8r11biqEl4gKAyDcsqjyJcwkgu8hEivJoc9FQqUtGpWaNdpzVc",
+        Description = "80 characters (maximum)")]
+    public void ValidPassword(string password)
```

Comment:
Why are these tests in a separate file from the other password tests?

Reply:
There are tests in other files? I don't see them.

Reply:
If you're referring to the registration tests; I do agree there's some overlap, but functionally, those are endpoint tests, while UserPasswordRuleTests are unit tests for the password validator. I think it's fine to keep these files separate, and there's no need to consolidate them.